backend/assets/js/aml.hljs.ts

```
@@ -4,7 +4,7 @@ import {HLJSApi, Language} from "highlight.js"
 const entityRegex = /^[a-zA-Z_][a-zA-Z0-9_#]*/
 const attributeNameRegex = /^ +[a-zA-Z_][a-zA-Z0-9_#]*/
 const attributeTypeRegex = /\b(uuid|(var|n)?char2?|character( varying)?|(tiny|medium|long|ci)?text|(tiny|small|big)?int(eger)?(\d+)?|numeric|float|double( precision)?|bool(ean)?|timestamp( with(out)? time zone)?|date(time)?|time( with(out)? time zone)?|interval|json|string|number)\b/
-const notesRegex = /\|[^#\n]*/
+const notesRegex = /(\|\|\|[^]*?\|\|\|)|(\|[^#\n]*)/
```

Biome

[error] The regular expression includes this negated empty character class.

Negated empty character classes match anything.
If you want to match against [, escape it \[.
Otherwise, remove the character class or fill it.

(lint/correctness/noEmptyCharacterClassInRegex)